### DISCUSSION TITLE
European Trip

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#1017] [General]
Hey guys! Really need your recommendations here. I plan visiting few different countries in Europe this summer and was curious what is the best way to get around there. Would you recommend public transport for example?

**Comment by MirkosMangle** [comment]
I lived in Germany and Switzerland before and must admit that train system is great in both countries.

**Comment by hernessteve** [comment]
I would recommend renting a car. It is not pricey at all. The European vignette store online [https://vignettestore.com/](https://vignettestore.com/) is the best place to get your toll passes before your trip. It’s fast and convenient. As someone who loves to travel by car, I've used the European vignette store online for multiple road trips across Europe.
